client/assets/stylesheets/reader-mobile.scss

```diff
@@ -0,0 +1,6 @@
+// CSS used by both WordPress iOS and Android to display Reader Content
+// Adding styles here will only affect the apps.
+//
+@import 'shared/utils';
+
+@import 'blocks/reader-full-post/style';
```

Comment:
Do we need the entire stylesheet here or just `'blocks/reader-full-post/_content`, which formats full post content?